src/Util/ArrayTwoOrMore.ts

```diff
@@ -0,0 +1,5 @@
+export type ArrayTwoOrMore<T> = [T, T] & T[];
```

Comment:
Is this util ever used? For me this looks something for the `base-util` package.

Reply:
Yes it is used by the LayerSwitcher.